publishable/config/voyager.php

```diff
@@ -132,17 +132,17 @@
     'dashboard' => [
         // Add custom list items to navbar's dropdown
         'navbar_items' => [
-            'Profile' => [
+            'voyager::generic.profile' => [
```

Comment:
The same changes you made to this file also need to be done in `voyager_dummy.php`

Reply:
Done